platform-core-module/src/main/java/com/softicar/platform/core/module/file/smb/jcifs/JcifsSmbClient.java

```diff
@@ -1,39 +1,44 @@
 package com.softicar.platform.core.module.file.smb.jcifs;
 
+import com.softicar.platform.common.core.exceptions.SofticarException;
 import com.softicar.platform.core.module.file.smb.ISmbClient;
 import com.softicar.platform.core.module.file.smb.ISmbDirectory;
 import com.softicar.platform.core.module.file.smb.ISmbFile;
 import com.softicar.platform.core.module.file.smb.SmbCredentials;
-import jcifs.Config;
-import jcifs.smb.NtlmPasswordAuthentication;
+import java.util.Properties;
+import jcifs.CIFSContext;
+import jcifs.CIFSException;
+import jcifs.config.PropertyConfiguration;
+import jcifs.context.BaseContext;
+import jcifs.smb.NtlmPasswordAuthenticator;
 
 public class JcifsSmbClient implements ISmbClient {
```

Comment:
I think this class should now be called `JcifsNgSmbClient`. Package should be `com.softicar.platform.core.module.file.smb.jcifsng`

Same goes for other classes in this package.